src/test/java/com/google/api/codegen/testdata/java_test_library.baseline

```diff
@@ -637,7 +631,7 @@ public class LibraryServiceTest {
     mockLibraryService.setResponses(expectedResponses);
 
     String formattedName = LibraryServiceApi.formatBookName("[SHELF]", "[BOOK]");
-    String indexName = "indexName746962392";
+    String indexName = "default index";
```

Comment:
Yes good point. I am not sure about the best way to resolve this. At the moment this value is coming from the sample_code_init_fields in the library_gapic.yaml file. It is showing up now because I changed the ordering of the check InitCodeGenerator. Do you think it is best to undo that change, and find a different way to set the initial value of the page token? Or should I change the test generator to not load from the sample_code_init_fields, and just use the required fields? Note that the previous surface is using "default_key" in the indexMap field, which is from the sample_code_init_fields.